Reloj_checador/app/assets/stylesheets/home.scss

```diff
@@ -7,7 +7,8 @@ html,body{
     font-family: 'Numans', sans-serif;
     }
     
-    .container2{
+    .login-container{
+    margin-top: 10rem;
```

Comment:
I recommend that you put the properties of your styles alphabetically, it is a good practice.